src/blip/Message.cc

```diff
@@ -343,7 +343,7 @@ namespace litecore { namespace blip {
 
     Error MessageIn::getError() const {
         if (!isError())
-            return {};
+            return {nullslice, 0, nullslice};
```

Comment:
I didn't like it either, but there was an error.  MSVC can't handle the fact that there is a `const int` there for whatever reason and a compilation error is raised about the inability to use the aggregate constructor.